editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -364,16 +365,17 @@ export const useDelayedCurrentStrategy = () => {
   return useDelayedEditorState<CanvasStrategyId | null>(selector)
 }
 
-const notResizableControls: ControlWithKey = {
+const notResizableControls = controlWithProps({
   control: NonResizableControl,
+  props: {},
   key: 'not-resizable-control',
   show: 'visible-except-when-other-strategy-is-active',
-}
+})
 
 export function getApplicableControls(
   currentStrategy: CanvasStrategyId | null,
   strategy: CanvasStrategy,
-): Array<ControlWithKey> {
+): Array<ControlWithProps<any>> {
```

Comment:
Please can we use `unknown` rather than `any` where possible

Reply:
let me see!